stag-library-compiler/src/main/java/com/vimeo/stag/processor/StagProcessor.java

```diff
@@ -104,27 +114,27 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                 Set<Modifier> modifiers = variableElement.getModifiers();
                 if (modifiers.contains(Modifier.FINAL)) {
                     throw new RuntimeException("Unable to access field \"" +
-                                               variableElement.getSimpleName().toString() + "\" in class " +
-                                               variableElement.getEnclosingElement().asType() +
-                                               ", field must not be final.");
+                                                variableElement.getSimpleName().toString() + "\" in class " +
+                                                variableElement.getEnclosingElement().asType() +
+                                                ", field must not be final.");
                 } else if (modifiers.contains(Modifier.PRIVATE)) {
                     throw new RuntimeException("Unable to access field \"" +
-                                               variableElement.getSimpleName().toString() + "\" in class " +
-                                               variableElement.getEnclosingElement().asType() +
-                                               ", field must not be private.");
+                                                variableElement.getSimpleName().toString() + "\" in class " +
+                                                variableElement.getEnclosingElement().asType() +
+                                                ", field must not be private.");
                 }
 
                 Element enclosingClassElement = variableElement.getEnclosingElement();
                 TypeMirror enclosingClass = enclosingClassElement.asType();
 
-                if (!TypeUtils.isParameterizedType(enclosingClass) ||
-                    TypeUtils.isConcreteType(enclosingClass)) {
-                    mSupportedTypes.add(enclosingClass.toString());
+                if (TypeUtils.isParameterizedType(enclosingClass) ||
+                        TypeUtils.isConcreteType(enclosingClass)) {
```

Comment:
I don't think this check is necessary anymore, since the `isParameterizedType` call checks whether the type is parameterized, and the `isConcreteType` checks whether either the class has no type parameters, or the type parameters that are there are resolved to real types.